docs/ansible/roles/grafana/index.rst

```diff
@@ -0,0 +1,24 @@
+.. _debops.grafana:
+
+debops.grafana
+==============
+
+The ``debops.grafana`` role can be used to install Grafana on a Debian or
+Ubuntu host. Grafana will be installed from upstream APT repository.
+
+.. toctree::
+   :maxdepth: 2
+
+   getting-started
+   defaults/main
```

Comment:
There's no `defaults-detailed` document that explains the `grafana__*_configuration` variables. Also, the documentation was rearranged a bit to add support for manual pages, perhaps you could update it here as well?